pallets/avn/src/lib.rs

```diff
@@ -408,8 +417,10 @@ impl<T: Config> OneSessionHandler<T::AccountId> for Pallet<T> {
             }
         });
 
+        let bounded_active_avn_validators = WeakBoundedVec::force_from(active_avn_validators);
```

Comment:
This could go inside the if in line 422, to optimise a little in case the list of collator hasn't changed (vast majority of the cases)